public/pages/index.html

```diff
@@ -0,0 +1,155 @@
+                <span class="welcome-place">@ National Museum of Korea, Art Center nabi and more</span>
+            </div>
+        </div>
+       
+    </section>
+      <section class="main-program" id="main-program">
+        <h2>Main Program</h2>
+        <div class="underline">&nbsp;</div>
+        <ul class="main-program-ul">
+            <li class="main-program-ul-li">
+            <div class="main-program-ul-li-div">
+                <div class="img-container main-program-ul-li-div-div"><img src="../resources/lecture.png" class="img-logo" alt="lecture">
+                    </div> 
+                    <h3 class="main-program-ul-li-div-h3">Lecture</h3>
+                    <p class="main-program-ul-li-div-p"> Listen to the spakers from various countries about the messages of sharing and opening</p>
+                </div>  </li>
+            
+            <li class="main-program-ul-li">
+                <div class="main-program-ul-li-div">
+                <div class="img-container main-program-ul-li-div-div"><img src="../resources/exhibition.png" class="img-logo" alt="ccExhibition">
+                </div> 
+                <h3 class="main-program-ul-li-div-h3">CC Exhibition</h3>
+                <p class="main-program-ul-li-div-p">Some artists and bands will be performing their songs in the show and talk about things.</p>
+            </div>
+            </li> 
+             <li class="main-program-ul-li">
+                <div class="main-program-ul-li-div">
+                    <div class="img-container main-program-ul-li-div-div"><img src="../resources/forum.png" class="img-logo" alt="forum">
+                    </div> 
+                    <h3 class="main-program-ul-li-div-h3">Forum</h3>
+                    <p class="main-program-ul-li-div-p">Here are some avaliable forums that can be useful and asks questions: Link</p>    
+                </div>
+
+            </li>  
+            <li class="main-program-ul-li">
+                <div class="main-program-ul-li-div">
+                <div class="img-container main-program-ul-li-div-div"><img src="../resources/workshop.png" class="img-logo" alt="workshop">
+                </div> 
+                <h3 class="main-program-ul-li-div-h3">Workshop</h3>
+                <p class="main-program-ul-li-div-p">Listen to the spakers from various countries about the messages of sharing and opening</p>
+                </div>
+            </li>
+              <li class="main-program-ul-li">
+                <div class="main-program-ul-li-div">
+                <div class="img-container main-program-ul-li-div-div"><img src="../resources/ignite.png" class="img-logo" alt="cc_ignite">
+                </div> 
+                <h3 class="main-program-ul-li-div-h3">CC Ignite</h3>
+                <p class="main-program-ul-li-div-p">Listen to the spakers from various countries about the messages of sharing and opening</p>
+            </div>
+        </li>
```

Comment:
## Required

- [ ] Please know that when you hover on a card a white border should appear as per the design:
  ![image](https://user-images.githubusercontent.com/20567503/196687661-1a1e7526-7cfe-4914-9785-150c4fad4419.png)
  In your case this is what happens:
  ![image](https://user-images.githubusercontent.com/20567503/196687865-c9ec9f07-18bf-4688-9ba3-ae8f9ab6bac0.png)
  Kindly fix this 😁.

## Optional

- [ ] For semantic reasons please know that each card should be inside an `article` tag, I suggest you change that as well 😁.